docs/topics/data-analysis/kotlin-notebook-set-up-env.md

```diff
@@ -0,0 +1,33 @@
+[//]: # (title: Set up an environment)
+
+<microformat>
+   <p>This is the first part of the <strong>Getting started with Kotlin Notebook</strong> tutorial:</p>
+   <p><img src="icon-1.svg" width="20" alt="First step"/> <strong>Set up an environment</strong><br/>
+      <img src="icon-2-todo.svg" width="20" alt="Second step"/> Create a Kotlin Notebook<br/>
+      <img src="icon-3-todo.svg" width="20" alt="Third step"/> Add dependencies to a Kotlin Notebook<br/>      
+      <img src="icon-4-todo.svg" width="20" alt="Fourth step"/> Share a Kotlin Notebook<br/>
+  </p>
+</microformat>
+
+Before you create your first Kotlin Notebook, you need to set up an environment.
+
+## Set up the environment
+
+1. Download and install the latest version of [IntelliJ IDEA Ultimate](https://www.jetbrains.com/idea/download/index.html).
+2. In IntelliJ IDEA, select **IntelliJ IDEA | Settings | Plugins** or **File | Settings | Plugins**.
+3. In the **Marketplace** tab, browse the **Kotlin Notebook** plugin and install it.
+
+![Install Kotlin Notebook from the marketplace](kotlin-notebook-plugin.png){width=700}
```

Comment:
I think the frame can be smaller so that rounding corners are not visible. Plus, the image should be indented to fit under step 3